widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -491,31 +475,13 @@ internal class ChatController(
 
     fun onResume() {
         Logger.d(TAG, "onResume")
-        if (hasPendingSurveyUseCase.invoke()) {
-            shouldHandleEndedEngagement = true
-            surveyUseCase.registerListener(this)
-            return
-        }
-        if (shouldHandleEndedEngagement) {
-            // Engagement has been started
-            if (!isOngoingEngagementUseCase.invoke()) {
-                // Engagement has ended
-                surveyUseCase.registerListener(this)
-            } else {
-                // Engagement is ongoing
-                onResumeSetup()
-            }
-        } else {
-            // New session
-            onResumeSetup()
-        }
+        onResumeSetup()
```

Comment:
There is no need in all these conditions anymore, is this correct?

Reply:
Yes, that's the reason I started these changes)